modules/apps/commerce/commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/security/resource/permission/CommerceVirtualOrderItemPermissionUtil.java

```diff
@@ -10,28 +10,21 @@
 import com.liferay.portal.kernel.security.permission.PermissionChecker;
 import com.liferay.portal.kernel.security.permission.resource.ModelResourcePermission;
 
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Alessio Antonio Rendina
  */
-@Component(service = CommerceVirtualOrderItemPermission.class)
-public class CommerceVirtualOrderItemPermission {
+public class CommerceVirtualOrderItemPermissionUtil {
 
-	public boolean contains(
+	public static boolean contains(
```

Comment:
remove this classe and call the `commerceVirtualOrderItemModelResourcePermission#contains` directly.